cmd/lingo/main.go

```diff
@@ -188,6 +193,11 @@ func run() error {
 		setupLog.Info("manager stopped")
 	}()
 
+	_ = mgr.GetCache().WaitForCacheSync(ctx)
```

Comment:
The mgr client will fail when the cache is not synced. Alternatively a client without cache could be passed.

Reply:
We should add an error check here right?